blamo/lib/Export/CSVExporter.dart

```diff
@@ -95,6 +165,177 @@ class CSVExporter {
+          + scrubData(tests[i].plasticLimit)
+          + scrubData(tests[i].fines)
+          + scrubData(tests[i].blows1)
+          + scrubData(tests[i].blows2)
+          + scrubData(tests[i].blows3)
+          + scrubData(tests[i].blowCount)
+          + "\n\n"
+      );
+    }
+  }
+
+  int buildProjectCSVController(){
+    buildProjectSegment();
+    buildPointSegment();
+    buildRemarksSegment();
+    buildWaterLevelsSegment();
+    buildWellSegment();
+    buildLithologySegment();
+    buildSampleSegment();
+  }
+
+  String buildProjectSegment() {
+    PROJECT.add(
+        scrubData(logInfo.project)
+      + "," //Name
+      + scrubData(logInfo.highway)
+      + scrubData(logInfo.loggedBy)
+      + scrubData(logInfo.county)
+      + scrubData(logInfo.number)
+      + "," //Input Units
+      + "," //Output Units
+      + "," //Depth Log page
+      + "," //Gap
+    );
+  }
+
+  String buildPointSegment() {
+    POINT.add(
+        scrubData(logInfo.project)
+      + scrubData(logInfo.north)
+      + scrubData(logInfo.east)
+      + scrubData(logInfo.equipment)
+      + scrubData(logInfo.startDate)
+      + scrubData(logInfo.endDate)
+      + "," //Purpose
+      + "," //Driller
+      + scrubData(logInfo.loggedBy)
+      + "," //Hole Depth
+      + "," //Key_No
+      + "," //Start_Card_No
+      + "," //Bridge_No
+      + scrubData(logInfo.surfaceElevation)
+      + "," //Depth Log
+      + scrubData(logInfo.tubeHeight)
+      + "," //Plunge
+      + "," //Gap
+    );
+  }
+
+  String buildRemarksSegment() {
+    REMARKS.add(
+        scrubData(logInfo.project)
+      + "," //Depth
+      + "," //Description
+      + "," //Show_Pointer
+      + "," //Gap
+    );
+  }
+
+  String buildWaterLevelsSegment() {
+    WATER_LEVELS.add(
+        scrubData(logInfo.project)
+      + "," //DateTime
+      + "," //Depth
+      + "," //Note
+      + "," //Gap
+    );
+  }
+
+  String buildWellSegment() {
+    WELL.add(
+        scrubData(logInfo.project)
+      + "," //Depth
+      + "," //Bottom
+      + "," //Graphic
+      + "," //Gap
+    );
+  }
+
+  String buildLithologySegment() {
+    for(int i = 0; i < units.length; i++){
+      LITHOLOGY.add(
+          scrubData(logInfo.project)
+        + scrubData("${units[i].depthLB}")
+        + scrubData("${units[i].depthUB}")
+        + "," //Grapic
+        + scrubData(formatTags(units[i].tags))
+        + "," //Formation
+        + "," //Gap
+      );
+    }
+  }
+
+  String buildSampleSegment() {
+    for(int i = 0; i < tests.length; i++){
+      SAMPLE.add(
+          scrubData(logInfo.project)
+        + scrubData("${tests[i].beginTest}")
+        + scrubData("${tests[i].endTest}")
+        + scrubData(tests[i].testType)
+        + scrubData(i.toString())
+        + scrubDataDouble(tests[i].percentRecovery)
+        + scrubData(tests[i].soilDrivingResistance)
+        + "," //N_value
+        + scrubData(tests[i].rockDiscontinuityData)
+        + "," //Percent Moisture
```

Comment:
Moisture content probably can go here

Reply:
I already used moisture content. There are two different fields one for percentage, and one for content.